src/gui/mzroll/point.cpp

```diff
@@ -141,9 +141,8 @@ void EicPoint::mousePressEvent (QGraphicsSceneMouseEvent* event) {
 
     if (_group) Q_EMIT peakGroupSelected(_group);
     if (_peak)  Q_EMIT peakSelected(_peak);
-
-    _mw->groupRtWidget->plotGraph(_group);
-
+    if (_group)
```

Comment:
Instead of doing a null check every time a method is called, it might be better to move this within the method. What do you think?

Reply:
I think it should be in both the places. The method or the caller, neither should assume that the group is a valid one.